src/06-positron-rag/lib/insert/prepareArticlesCollection.ts

```diff
@@ -0,0 +1,141 @@
+  const client = weaviate.client({
+    scheme: "https",
+    host: "https://weaviate.stg.artsy.systems",
+  })
+
+  const alreadyExists = await client.schema.exists(CLASS_NAME)
+
+  if (alreadyExists) {
+    console.log(`${CLASS_NAME} class already exists, deleting it`)
+    await client.schema.classDeleter().withClassName(CLASS_NAME).do()
+  }
+
+  const classWithProps = {
+    class: CLASS_NAME,
+    vectorizer: "text2vec-openai",
+    moduleConfig: {
+      "text2vec-openai": {
+        model: "text-embedding-3-small",
+        dimensions: 1536,
+        type: "text",
+        vectorizeClassName: false,
+      },
+    },
+    properties: [
+      {
+        name: "internalID",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "title",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "href",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "publishedAt",
+        dataType: ["date"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "byline",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      // {
+      //   name: "keyword",
+      //   dataType: ["text"],
+      //   moduleConfig: {
+      //     "text2vec-openai": {
+      //       skip: true,
+      //       // vectorizePropertyName: false,
+      //     },
+      //   },
+      // },
+      {
+        name: "vertical",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "channelName",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "head",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: false,
+            // vectorizePropertyName: false,
+          },
+        },
+      },
+      {
+        name: "body",
+        dataType: ["text"],
+        moduleConfig: {
+          "text2vec-openai": {
+            skip: true, // TODO: vectorize separately?
+            // vectorizePropertyName: false,
+          },
+        },
```

Comment:
For now, `body` (the plaintext of the full article content) is stored but _not_ vectorized. 

I wanted to work with the simpler  `head` first.

(`head` is a plaintext concatenation of `title`, `byline`, `description` and `keywords`)

Reply:
> For now, body (the plaintext of the full article content) is stored but not vectorized.

Any specific reason for not wanting to vectorize the `body`? Is it you just don't need it right now, so why vectorize?

Reply:
Yeah, as well as my suspicion that vectorizing smaller chunks would yield better results. But that's something to confirm experimentally.